pkg/daemon/osrelease.go

```diff
@@ -44,6 +44,15 @@ func (os OperatingSystem) IsLikeTraditionalRHEL7() bool {
 	return os.VersionID == "7"
 }
 
+// IsLikeTraditionalRHEL8 is true if the OS is traditional RHEL8 or CentOS8:
+func (os OperatingSystem) IsLikeTraditionalRHEL8() bool {
```

Comment:
Also this new func doesn't seem to be called anywhere not in a test? bc to support rhel8 i'd expect an update to update.go/daemon.go as well?